src/ScottPlot/Plot.cs

```diff
@@ -1089,7 +1089,7 @@ public void Clear(Predicate<Plottable> plottablesToClear)
 
             settings.plottables.Add(barPlot);
 
-            if (autoAxis)
+            if (autoAxis && !ys.Where(y => y < 0).Any())
```

Comment:
Is this intentional? I'll probably remove it... my thinking is that if the user set the `autoAxis` argument to `true`, `AxisAuto()` should get called regardless of Y values

Reply:
Scratch that - I see the issue now! When negative Y values are used, y2 (not y1) should be set to zero.